CHANGELOG.md

```diff
@@ -3,13 +3,18 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+
 v1.12.0
 ------------------------------
 *August 28, 2019*
 
 ### Fixed
 - Removed mask from logo SVG to stop IE11 issues not rendering the T of JustEat
 
+v1.11.3
```

Comment:
You should move your changes up with the latest version, v1.13.0 it will be and change the version in package.json too

Reply:
And also in the PR title so the merge commit message will be correct